README.md

```diff
@@ -156,6 +148,50 @@ Configure the form-submit functionality which enables you to submit a (login) fo
 | input_filter      | A list of input fields that should not be submitted with the form                                         | False    |         | string - list       |
 | submit_once       | Submit the form only once on startup instead of each scan interval                                        | False    | False   | boolean             |
 | resubmit_on_error | Resubmit the form after a scraping error is encountered                                                   | False    | True    | boolean             |
+| variables         | See [Form Variables](#Form-Variables)                                                                     | False    |         | list                |
+
+### Form Variables
+
+Configure the variables you want to be forwarded from scraping the [Form-submit](#form-submit) page to scraping the main page for sensor data. A common use case is to populate the `X-Login-Token` header which is the result of the login.
```

Comment:
I propose:
Configure the variables that will be scraped from the `form_submit` response. You  will be able to use those values in the `value_template` of a `header` or a `selector` in the main configuration. A common use case is to populate the `X-Login-Token` header which is the result of the login.

Reply:
I see now that you also use the variables in the request data and params templates. Let's mention that as well here.

Reply:
I'm unsure how to phrase this. Can you suggest, please?

Reply:
Let's continue this discussion in the other comment.